### ISSUE TITLE
Overlays are not accessible

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#257]
![image](https://cloud.githubusercontent.com/assets/7644740/4441424/a1776bee-47d2-11e4-91e9-35f789d13294.png)
